packages/hash/frontend/src/pages/_app.page.tsx

```diff
@@ -119,6 +117,19 @@ const App: FunctionComponent<AppProps> = ({
           </ModalProvider>
         </ThemeProvider>
       </CacheProvider>
+      {/* eslint-disable-next-line react/no-unknown-property -- leftover from tailwind; can be removed when `animation: spin` is removed from JSX CSS */}
+      <style jsx global>
```

Comment:
You can use the GlobalStyles component from MUI instead

Reply:
Good point. Switched here: https://github.com/hashintel/hash/pull/1471/commits/c15e23c60f9214eaf1ef073cc9a4bc101bb374ad